packages/typespec-ts/test/modularIntegration/azureCoreTraits.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { TraitsClient } from "./generated/azure/core-traits/src/index.js";
+import { assert } from "chai";
+
+describe("Traits Client", () => {
+  let client: TraitsClient;
+
+  beforeEach(() => {
+    client = new TraitsClient({
+      allowInsecureConnection: true
+    });
+  });
+
+  it("client smoke test", async () => {
+    try {
+      const result = await client.smokeTest(1, "123", {
+        ifNoneMatch: '"invalid"',
```

Comment:
The value of `ifNoneMatch` is set to `'"invalid"'` and `ifMatch` is set to `'"valid"'` in mock file. They have an extra pair of single quotes than normal `string` type. Please have a check if there are any issues here.

Reply:
I think this is by design and etag value is quoted string https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag#examples.